src/description/eth_call.md

```diff
@@ -0,0 +1,15 @@
+* **MUST NOT** affect on-chain state 
+
+* **MUST** accept a `from` account that does not exist on-chain and if `from` is not defined it **MUST** be set to `0x0000000000000000000000000000000000000000`
```

Comment:
I'm not a huge fan of the "set" terminology here.  Consider instead "**MUST** be interpreted as ..." instead?  The receiver isn't "setting" anything, they are reading/interpreting incoming data.

Reply:
I had to read this sentence several times, it first looked to me like eth-call must accept _only_ from-addresses that do not exist.